chaincode/output.go

```diff
@@ -199,16 +199,17 @@ func (outputTraintuple *outputTraintuple) Fill(db LedgerDB, traintuple Traintupl
 }
 
 type outputTesttuple struct {
-	Key       string         `json:"key"`
-	Algo      *HashDressName `json:"algo"`
-	Certified bool           `json:"certified"`
-	Creator   string         `json:"creator"`
-	Dataset   *TtDataset     `json:"dataset"`
-	Log       string         `json:"log"`
-	Model     *Model         `json:"model"`
-	Objective *TtObjective   `json:"objective"`
-	Status    string         `json:"status"`
-	Tag       string         `json:"tag"`
+	Key           string         `json:"key"`
+	Algo          *HashDressName `json:"algo"`
+	Certified     bool           `json:"certified"`
+	Creator       string         `json:"creator"`
+	Dataset       *TtDataset     `json:"dataset"`
+	Log           string         `json:"log"`
+	TraintupleKey string         `json:"traintupleKey"`
+	Model         *Model         `json:"model"` // Obsolete field. TODO: delete it
```

Comment:
If it's obsolete, it should be removed no ?

Reply:
It was but put back to guarantee backward compatibility. But as far as we know, nothing breaks if it's removed. We intend to do so in a future PR.

Reply:
@thibaultrobert We do backward compatibility on the chaincode ?
I thought we need a upgrade function before dealing with backward compatibility on the chaincode, no ?

Reply:
You right about that. We do not support chaincode update for now. Sorry if my statement was misleading.
The backward compatibility is not between two chaincode version but between the two different we deal with models, the ones before and after this PR. As I say the PR https://github.com/SubstraFoundation/substra-backend/pull/33 should ensure that deleting the `model` field won't have side effects but we wish to invest more time be sure. So we keep both approach for now and we'll come back to it later.

Reply:
Okay ! Thank you for those information. I put also a global comment on my review that it may have an impact (I did not test it) on substra-frontend :)